addons/web/static/src/core/dialog/dialog.scss

```diff
@@ -1,52 +1,12 @@
-.o_dialog {
-  > .modal {
-    display: block;
+// This code should be removed and standard utility classes should be used instead
+.modal.o_technical_modal .modal-content .modal-footer footer {
+  > :not(:first-child) {
+    margin-left: map-get($spacers, 1);
```

Comment:
This scss will need to be removed and standard utility classes used instead after BTS5 implementation